website_multi_theme/data/ir_module_category.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2018 Ivan Yelizariev <https://it-projects.info/team/yelizariev>
+     License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html). -->
+<odoo>
+
+    <record model="ir.module.category" id="base.module_category_theme">
+        <field name="exclusive" eval="False"/>
```

Comment:
I can't believe it was so simple! Could you backport this to v10? :thinking: Or was it hardcoded in v10?

Reply:
Yes, it's hardcoded and the only way is rewrite (not inherit) one method